suite-native/intl/src/en.ts

```diff
@@ -188,6 +188,15 @@ export const en = {
             accountLabelBadge: 'Run on {accountLabel}',
         },
     },
+    transactions: {
+        caution: 'Caution!',
+        detail: {
+            warning: {
+                title: 'Caution!',
```

Comment:
This translated string is duplicated. Maybe create it just once?

Reply:
correct. They are the same

I've been thinking about this and my approach is this one:

There might be technically multiple duplicates of some texts. Those texts might be used in different contexts, sometimes as CTA, sometimes as Title etc. But once translations are done on those texts according to the context or language nuances, the texts in the other translations can differ (and in my experience they sometimes do). For that reason, if texts are used in different contexts, they should/can be duplicate. This situation here is somewhere on the edge between the same context and different one, so I was not 100% sure what approach to take.

Wdyt? Let's discuss! :)

Reply:
I think you are right. The context might change... 

So maybe let's don't use `caution` as a translation string id, and make it whole more comprehensive?
eg:
```
phishing: {
badgeLabel: "Caution!"
alertTitle: "Caution!",
alertContent: "..."

}
```

What do you think?

Reply:
Answer it as always it depends. I will leave this up to you guys, if you think that context might change feel free to duplicate that, but also try to avoid unneccasary duplications if possible.

Also it's worth mentioning that it's very easy to just replace that one usage with changed context with more specific variant than doing opposite.

Also I agree with @PeKne about more general translations ids that are not that tied to it's content, but again it depends...

Reply:
09ace9f